controls/V-72317.rb

```diff
@@ -63,4 +72,14 @@
 finding."
   tag "fix": "Remove all unapproved tunnels from the system, or document them with
 the ISSO."
+
+    @conn_grep_results = inspec.command("grep -i issue /etc/audit/audit.rules").stdout.split("\n")
+
+    @conn_grep_results.each do |curr_line|
+        describe curr_line do
+          it { should be_in tunnels }
```

Comment:
Checking for `conn` in the rules won't work because this is just "how it works".

Since this is an "unauthorized" finding, it has to always fail unless someone provides input as to what tunnels it's actually expecting. 

We configure tunnels with Puppet in SIMP so I'm 100% sure that all other tunnels are purged. Otherwise, this is just a catch 22 check where you *always* have to ask if what is found is OK.

Reply:
The audit rules thing was a mistake. I switched it to checking in the config files as the STIG mentions. The "tunnels" attribute will have to be populated with the expected tunnels so when an unexpected one is found it will fail.